src/js/no-react-entry.js

```diff
@@ -1,6 +1,8 @@
 import createCommonStore from './common/store';
 import createLoginWidget from './login/login-entry';
 
+const wizardPages = new Set(['/education/apply/', '/education/eligibility/']);
```

Comment:
@cehsu @cvalarida Hello! Looks like the wizard is enabled. Is this PR ready to be merged?